src/Makefile.am

```diff
@@ -69,6 +69,14 @@ libephymain_la_SOURCES = \
 	ephy-session.h				\
 	ephy-shell.c				\
 	ephy-shell.h				\
+	ephy-sync-crypto.c			\
```

Comment:
There's enough files here, I think it would make sense to move them into their own subdirectory.